packages/@atjson/renderer-commonmark/src/index.ts

```diff
@@ -1,43 +1,99 @@
 import { Bold, Code, HTML, Heading, Image, Italic, Link, List } from '@atjson/offset-annotations';
 import Renderer, { Context } from '@atjson/renderer-hir';
+import { Annotation } from '@atjson/document';
+
+const WHITESPACE_PUNCTUATION = /((\s|&nbsp;){1}|(([\u2000-\u206F]|\\?[!-\/:-@\[-`{-~])))/;
+const BEGINNING_WHITESPACE_PUNCTUATION = new RegExp(`^${WHITESPACE_PUNCTUATION.source}`);
+const ENDING_WHITESPACE_PUNCTUATION = new RegExp(`${WHITESPACE_PUNCTUATION.source}$`);
+
+function getPreviousChar(content: string, index: number) {
+  let idx = index - 1;
+  let char = content[idx];
+  while (char && char.charCodeAt(0) === 65532) {
```

Comment:
maybe set 65532 to a const? Just so that its more clear what char are we looking for

Reply:
Eh, can we change this to `char === '\uFFFC'`

Reply:
👍